R/expect_null_linter.R

```diff
@@ -0,0 +1,43 @@
+#' @describeIn google-linters Require usage of expect_null(x) over expect_equal(x, NULL) and similar
```

Comment:
Is there a way to describe these both in google-linters and in linters?
IMO there should be one help page that lists all available linters.

Reply:
Agree it makes sense to have one landing page, but maybe we need

```
+ available linter suites
└-- linters
└-- google-linters
└-- ...
```

instead? ?linters is already 300 lines long

Reply:
Opened #888 for discussion around this.
Maybe we could also export a function that returns a tibble of all available linters?
We could add some metadata to this and would enable using R to analyze the available sets more quickly than skimming a 300 lines long man page...

Reply:
sounds good! Do we want to handle that as part of this PR? or handle it later?

Reply:
I think we should discuss this first to minimize the need for refactoring, but handle necessary changes in a separate PR.